test/integration/command/test_audit.py

```diff
@@ -36,7 +36,7 @@ def test_conan_audit_paths():
                                 "description": "Zip vulnerability",
                                 "severity": "Critical",
                                 "cvss": {
-                                    "preferredBaseScore": 9.8
+                                    "preferredBaseScore": 8.9
```

Comment:
Changed it here because the default severity level raises an error now, causing regression for this specific test.